File: DAY 1-java basics/gcd.java
import java.util.*;
public class gcd{
    public static void main (String[] args){
        Scanner sc = new Scanner(System.in);
        System.out.println("enter the two numbers: ");
        int a = sc.nextInt();
        int b = sc.nextInt();
        for(int i =b ; i>0; i--){
            if(a%i==0 && b%i==0){
                System.out.println("GCD is:" +i );
                break;
            }
        }   
    }
}

File: DAY 1-java basics/gcd.cpp
#include<iostream>
using namespace std;
int main(int argc , char** argv){
    int a ;
    int b ;
    cout<<"enter the two numbers: "<<endl;
    cin>>a>>b;
    while(b%a!=0){
        int c = b%a;
        b=a;
        a=c;

    }cout<<a;
}
